[*********************100%%**********************]  1 of 1 completed

1 Failed download:
['AAPL']: ValueError('day is out of range for month')
Traceback (most recent call last):
  File "/mnt/c/Users/RAIKA KOKI/B4研究/Multi_iTransformer/STLdemo.py", line 28, in <module>
    raise Exception("No data fetched for the given stock code and date range.")
Exception: No data fetched for the given stock code and date range.
Traceback (most recent call last):
  File "/mnt/c/Users/RAIKA KOKI/B4研究/Multi_iTransformer/STLdemo.py", line 28, in <module>
    raise Exception("No data fetched for the given stock code and date range.")
Exception: No data fetched for the given stock code and date range.
